.github/workflows/helm-lint-test.yaml

```diff
@@ -75,4 +75,15 @@ jobs:
 
       - name: Run chart integration tests
         if: steps.chart-list-changed.outputs.changed == 'true'
-        run: ct lint-and-install --config .github/configs/helm-lint-test.yaml
+        run: ct lint-and-install --config .github/configs/helm-lint-test.yaml --skip-clean-up
+      
+      - name: Print pod logs on failure
+        if: failure() && steps.chart-list-changed.outputs.changed == 'true'
+        run: |
+          echo "🔍 Printing logs for failed pods..."
+          for ns in $(kubectl get ns -o jsonpath='{.items[*].metadata.name}'); do
```

Comment:
We don't have to iterate over all the namespaces. The namespace will be always `default`. We have declared the namespace here -https://github.com/cloudoperators/greenhouse-extensions/blob/ci-test-pod-logs/.github/configs/helm-lint-test.yaml#L9